source/NVDAObjects/window/winword.py

```diff
@@ -994,9 +1022,13 @@ def _iterNodesByType(self,nodeType,direction="next",pos=None):
 			revisions=RevisionWinWordCollectionQuicknavIterator(nodeType,self,direction,rangeObj,includeCurrent).iterate()
 			return browseMode.mergeQuickNavItemIterators([comments,revisions],direction)
 		elif nodeType in ("table","container"):
-			 return TableWinWordCollectionQuicknavIterator(nodeType,self,direction,rangeObj,includeCurrent).iterate()
+			return TableWinWordCollectionQuicknavIterator(nodeType,self,direction,rangeObj,includeCurrent).iterate()

```

Comment:
This was, probably by accident, indented with two tabs and one space, I removed the space to bring this in line with other return statements here.